client/my-sites/stats/stats-date-picker/date-label-drill/index.tsx

```diff
@@ -19,12 +23,31 @@ const DateLabelDrill = ( { children }: DateLabelDrillProps ) => {
 		window.history.back();
```

Comment:
Note related to the PR: The back function here limited the ability to further dill up as otherwise it wouldn't act like an up button anymore. I don't see why not we do not user to further drill up if they drill down from there. I would suggest we remember the drill down path in the URL or in sessionStorage, and it could be used for the tooltip as well. 

cc/ @dognose24

Reply:
> I don't see why not we do not user to further drill up if they drill down from there.

I was wondering about this, too. It should be possible for us to drill up until the main date filter. I think we can handle this in a new task

Reply:
Keeping all the history paths would make the drill-up button more versatile. Sounds good to me! 👍🏼